assets/js/date-picker.js

```diff
@@ -0,0 +1,219 @@
+import datePicker from './bootstrap-datepicker1.8.0.min';
```

Comment:
We should create a date picker directory and then move all relevant files to it.